modules/3_helpernode/templates/helpernode_vars.yaml

```diff
@@ -92,3 +92,4 @@ ocp_install_kernel: "file:///dev/null"
 
 # This is required for latest helpernode. TODO: Remove when https://github.com/RedHatOfficial/ocp4-helpernode/pull/140 is merged
 helm_source: "https://get.helm.sh/helm-v3.4.0-linux-ppc64le.tar.gz"
+fips: ${fips}
```

Comment:
/hold
Looks like we will need to wait for this to be merged and use proper references.

Reply:
Yes, thanks, I made that clear in the description.

Reply:
https://github.com/redhat-cop/ocp4-helpernode/pull/320 is now merged